app/features/build/views/build.erb

```diff
@@ -56,7 +79,7 @@
     uri += "/?&project=<%= project.id %>&build=<%= build.number %>"; // we add that extra & to make the Ruby URI parser work
     var ws = new WebSocket(uri);
     console.log("Listening to web socket connection now for URL " + uri + "...")
-    var firstMessageReceived = <%= existing_rows.count == 0 ? false : true %>;
+    var firstMessageReceived = <%= existing_rows.to_s.length == 0 ? false : true %>;
```

Comment:
Could we use [`.zero?`](https://ruby-doc.org/core-1.9.3/Numeric.html#method-i-zero-3F) instead, negate and remove the ternary, please?

```ruby
!existing_rows.to_s.length.zero?
```

Reply:
Same as https://github.com/fastlane/ci/pull/498#discussion_r178723810

Reply:
@KrauseFx:
```
1.zero?
=> false
```

Reply:
it works on `Fixnums`

Reply:
Ah I misread the comment. I'm ok with keeping `length == 0`